lib/_debugger.js

```diff
@@ -126,7 +130,7 @@ Protocol.prototype.execute = function(d) {
       break;
 
     default:
-      throw new Error('Unknown state');
+      throw I18N.Error(I18N.NODE_DEBUGGER_UNKNOWN_STATE);
```

Comment:
One infinitesimal drawback to removing the `new` keyword is that it changes the top of the error stack to CreateError in lib/internal/messages.js, i.e., it changes the origin.